docs/Design/SoftDetailedDes/MIS.tex

```diff
@@ -229,6 +228,92 @@ \subsubsection{Local Functions}
 
 \newpage
 
+\section{MIS of Location Management Module} \label{mLocation} 
+The Location Management Module manages user location data and outputs relevant resources based the available user location data.
+
+\wss{You can reference SRS labels, such as R\ref{R_Inputs}.}
+
+\wss{It is also possible to use \LaTeX for hypperlinks to external documents.}
+
+\subsection{Module}
+
+locationManagement
+
+\subsection{Uses}
+
+ Database Driver Module (Section \ref{mDBDriver})
+
+\subsection{Syntax}
+
+\subsubsection{Exported Constants}
+None
+
```

Comment:
Not necessarily a change you need to make, but just noting that this is something I did so you can consider it for future MIS if it isn't applicable here.

I added a section here for Exported Types. I know it isn't necessary for everyone and not sure if you need to create a type for Locations, but I did something, but if it's not applicable for you, then it doesn't really matter.

![image](https://github.com/r-yeh/grocery-spending-tracker/assets/67233377/ce9c2f42-ead6-402e-8527-c90aad0a5b3a)
![image](https://github.com/r-yeh/grocery-spending-tracker/assets/67233377/db53f03d-8b36-4cd0-ad3c-92585be0972c)

Reply:
pretty sure i don't have any exported types

Reply:
Yeah just something to keep in mind for future MIS if you need it